examples/ROOT_files/ROOT_STIR_consistency/run_pretest_script.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/bash
+
+# Copyright (C) 2022, University College London
+#   This file is part of STIR.
+# SPDX-License-Identifier: Apache-2.0
+# See STIR/LICENSE.txt for details
+
+# Authors: Elise Emond and Robert Twyman
+
+#### Discription ####
+# This script generates STIR volumes and runs a GATE sumulation of 12 point sources.
+# This data is designed to be used by the `test_view_offset_root` test in STIR.
+
+
+# This script should be run from the directory containing this file `run_pretest_script.sh`
+if ! [ -f "run_pretest_script.sh" ]; then
+    echo "'run_pretest_script.sh' should be run from the directory containing the script."
+    exit 1
+fi
+
+# Create a directory for the STIR images
+PRE_TEST_OUTPUT_DIR=pretest_output
+mkdir -p ${PRE_TEST_OUTPUT_DIR}
+
+for I in {1..12}
+do
+	echo ""
+	echo "Generating data for test${I}..."
+	
+	# Generate images and move to the pretest output directory
+	echo "Generating STIR image..."
+	generate_image SourceFiles/generate_image${I}.par
+	mv stir_image${I}.ahv ${PRE_TEST_OUTPUT_DIR}
+	mv stir_image${I}.hv ${PRE_TEST_OUTPUT_DIR}
+	mv stir_image${I}.v ${PRE_TEST_OUTPUT_DIR}
```

Comment:
```suggestion
	mv stir_image${I}.*v ${PRE_TEST_OUTPUT_DIR}
```
simpler and more future proof